app/web/src/components/Hero/Hero.tsx

```diff
@@ -108,7 +127,13 @@ export const Hero = () => {
               projectTitle: 'coffee-lid',
             })}
           >
-            <div className="grid grid-flow-col gap-2 sm:gap-4 items-center bg-ch-gray-760 bg-opacity-95 text-ch-gray-300 rounded-md p-2 font-fira-sans relative z-10 shadow-ch">
+            <div
+              className="grid grid-flow-col gap-2 sm:gap-4 items-center bg-ch-gray-760 bg-opacity-95 text-ch-gray-300 rounded-md p-2 font-fira-sans relative z-10 shadow-ch"
+              style={{
+                transform: 'translate3d(3vw, -100px, 0.3px) scale(0.7)',
```

Comment:
bringing the element forward 0.3px to get parallax and scaling down so it's still the same size. (0.3px is enough because the perspective is only 1px).

Few position tweaks too on x and y